tests/phpunit/unit-tests/Helper/Fields/Test_Field_Select.php

```diff
@@ -57,6 +57,21 @@ public function test_is_empty() {
 		remove_filter( 'gfpdf_field_is_empty_value_instead_of_label', '__return_false' );
 	}
 
+	public function test_is_empty_falsey() {
```

Comment:
@jestonihpi you should also feed the test a real string like you would expect in a real form entry to verify `is_empty` isn't triggered incorrectly.